concourse/tasks/integration.yml

```diff
@@ -0,0 +1,61 @@
+version: "3.3"
```

Comment:
this file is in concourse/tasks/, which makes it seem like it should be a task, but it's a docker compose file? let's call it `docker-compose-fauna.yml`